docs/modules/ROOT/nav.adoc

```diff
@@ -7,6 +7,9 @@
 * xref:cluster-monitor.adoc[Couchbase Cluster Monitor component]
 * xref:deployment-distributed.adoc[Distributed microservices deployment]
 
+.Components
+* xref:component-prometheus.adoc[Prometheus]
```

Comment:
Are we going to link to this from a diagram or something? It doesn't seem to provide a huge level of value otherwise.

I'd suggest focusing on the Couchbase Server 7 improvements plus the cloud native existing support for Prometheus.

Reply:
I am not quite sure what you mean, the structure of pages is not clear to me. I am addressing [this ticket](https://issues.couchbase.com/browse/CMOS-69) for various components.

Reply:
Currently this page is just a bit of blurb about Prometheus which feels more like a Grafana press release than a Couchbase document. By all means we can have a little intro but it needs to focus on what it provides in the context of CMOS. 
We also want to highlight that CBS 7 has native Prometheus support too which we are aligning with and building on to provide a capable monitoring solution out of the box.

Reply:
The JIRA you linked includes scope for a diagram which was my original question: are we having a clickable diagram that then jumps to these pages? That might be a good idea and the landing pages can then call out the OSS docs.

Reply:
I am not sure how this is to be structure tbh. Adding a dedicated section should appear itself in the navigation pane.

This section links to the actual page which has a diagram.